Autopilot/TelemetryManager/Telemetry_Manager.c

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file Telemetry_Manager.c
+ * @author Jingting Liu
+ * @date January 14th, 2021
+ */
+
+#include <stdio.h>
+#include <common/mavlink.h>
+
+#include "telemetry_state_machine.h"
```

Comment:
I think this is an old filename for "Telemetry_Manager.hpp".

Reply:
nice catch, thanks!